docs/includes/apiargs-MongoDBDatabase-method-createCollection-option.yaml

```diff
@@ -35,6 +35,20 @@ pre: |
   </reference/bson-type-comparison-order/#collation>` for the collection.
 ---
 arg_name: option
+name: expireAfterSeconds
+type: integer
+description: |
+  Available on 5.0+ servers only
```

Comment:
Based on existing docs, this should be phrased as:

> This option is available in MongoDB 5.0+ and will result in an exception at execution time if specified for an older server version.

Alternatively (as is done later in this file for `writeConcern`):

> This is not supported for server versions prior to 5.0 and will result in an exception at execution time if used.

In both cases, these lines are typically added _after_ any documentation describing the option's functionality. For instance, the `writeConcern` docs later in this file add that using `post:`.

Reply:
I've updated the documentation as suggested. Thanks!